bin/diagnostic.js

```diff
@@ -1 +1,28 @@
+##Question 5
+
 'use strict';
+
+
+
+//We'll learn about require later in the course
+const ask = require('../lib/ask.js');
+
+let answer = 'not empty';
+
+const randNumbers = [1,2,3,4,5,6,7,8,9,10];
+
+while (answer !== '') {
+  answer = ask("Guess my secret? ");
+  if (answer = randNumbers) {
```

Comment:
Does this line validate and work? Not sure if it does because it would se answer = an array. Close though.